C/cargo_c/build_tarballs.jl

```diff
@@ -0,0 +1,63 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "cargo_c"
+version = v"0.10.14"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/lu-zero/cargo-c", "d1da3c27a0defdceba12c2b2762094cdde6c0337"),
+    FileSource("https://github.com/lu-zero/cargo-c/releases/download/v$(version)/Cargo.lock", "0af99496210d7e5cb70de6643b571a8882120148e628d7a75327937b64fd9841")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cp $WORKSPACE/srcdir/Cargo.lock $WORKSPACE/srcdir/cargo-c/
+cd $WORKSPACE/srcdir/cargo-c/
+cargo build --release --locked --features=vendored-openssl
+install -Dvm 755 "target/${rust_target}/release/cargo-capi${exeext}" "${bindir}/cargo-capi${exeext}"
+install -Dvm 755 "target/${rust_target}/release/cargo-cinstall${exeext}" "${bindir}/cargo-cinstall${exeext}"
+install -Dvm 755 "target/${rust_target}/release/cargo-cbuild${exeext}" "${bindir}/cargo-cbuild${exeext}"
+install -Dvm 755 "target/${rust_target}/release/cargo-ctest${exeext}" "${bindir}/cargo-ctest${exeext}"
+install_license LICENSE
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms()
+
+# Rust toolchain for i686 Windows is unusable
+filter!(p -> !Sys.iswindows(p) || arch(p) != "i686", platforms)
+
+# Rust toolchain 1.87.0 not available on platform aarch64-unknown-freebsd or riscv64-linux-gnu
+filter!(p -> !(arch(p) == "aarch64" && Sys.isfreebsd(p)), platforms)
+filter!(p -> !(arch(p) == "riscv64"), platforms)
+
+# Errors with "ld: library not found for -lclang_rt.osx"
```

Comment:
That's in llvm compiler rt

Reply:
For some reason, I still get this error even if I have `LLVMCompilerRT_jll`